### PR TITLE
Customer gets email on auction acceptance

### PR DESCRIPTION
**Pull request by jessieay**
* Closes https://github.com/18F/micropurchase/issues/779

* Note: we may add some more content to this later depending on Pcard (eg: non-18F pcard users are asked to pay and given credit card URL).

screen shot 2016-07-07 at 12 22 08 pm

### CODE REVIEW

**Comment by mtorres253** [comment]
@jessie Hi! added the bit about the delivery URL that we discussed

**Comment by jessieay** [comment]
@mtorres253 updated! new email copy:

screen shot 2016-07-07 at 1 24 50 pm

File: app/models/customer.rb [hunk 1]
```diff
@@ -1,7 +1,6 @@
-# Represents a single agency customer
 class Customer < ActiveRecord::Base
   validates :agency_name, presence: true, uniqueness: true
-  validates :email, email: true, presence: false
+  validates :email, email: true, allow_nil: true
```
Comment (harrisj) [inline app/models/customer.rb]: Ah thank you, I knew I was doing that wrong

**Comment by harrisj** [comment]
Looks good to me! Merging

File: app/models/customer.rb (hunk 1 above)
Comment (jessieay) [inline app/models/customer.rb]: I never would have known until I tried creating a customer without an email!

**Pull request merged**